components/skus/browser/rs/lib/Cargo.toml

```diff
@@ -22,7 +22,7 @@ tracing = { version = "0.1", default-features = false }
 bigdecimal = { version = "0.1", features = ["serde"] }
 serde = { version = "1.0", features = ["derive"] }
 chrono = { version = "0.4", default-features = false, features = ["clock", "serde"] }
-uuid = { version = "0.8", features = ["serde"] }
+uuid = { version = "0.8", features = ["v4", "serde"] }
```

Comment:
We're already shipping uuid v1 with the `v4` feature enabled. Can you use that instead?